module/zfs/dmu_traverse.c

```diff
@@ -39,7 +39,7 @@
 #include <sys/zfeature.h>
 
 int32_t zfs_pd_bytes_max = 50 * 1024 * 1024;	/* 50MB */
-int32_t ignore_hole_birth = 1;
+int32_t send_holes_without_birth_time = 1;
```

Comment:
In the description you said you changed it to a boolean_t, but I don't see that change here.